reports/report.Rmd

```diff
@@ -0,0 +1,77 @@
+
+
+## Introduction
+
+With the rising popularity of online shopping, particularly in the wake of the 2020 coronavirus pandemic, there exists a strong growth opportunity for businesses employing e-commerce solutions. While increasing overall traffic to online stores is a critical first step, higher traffic does not always convert into increased sales. Online visitors may exit the site without making a purchase for a variety of reasons including: loading times, website layout, pricing, and others.
+
+In this project, we attempt to use machine learning to predict whether a visitor of an online shopping website is intending on making a purchase based on session data that looks at the pages they visit, the duration of each visit, and the source of traffic among other factors. 
+
+Being able to predict purchasing intentions can be a valuable tool because it could potentially allow businesses to implement measures to better capture users with purchasing intentions and thus try to convert them into actual revenue generators. For example, users with purchasing intentions can be served with targeted content that aims to reduce shopping cart abandonment. Predicting purchasing intentions can also be useful for making more accurate forecasts about the business. A business with a given portfolio of visitors may use the machine learning algorithm to predict its revenue conversion rates which can be a valuable metric in financial modeling.
+
+## Methdology
+
+### Data
+
+The data set used in this project is the "Online Shoppers Purchasing Intention" dataset provided by the [Gözalan Group](http://www.gozalangroup.com.tr/) and used by Sakar et. al in their analysis published in [Neural Computing and Applications](https://link.springer.com/article/10.1007/s00521-018-3523-0) [@Sakar2019]. The data set was sourced from UCI's Machine Learning Repository [@Dua:2019] at this [link](https://archive.ics.uci.edu/ml/datasets/Online+Shoppers+Purchasing+Intention+Dataset). The specific file used for the analysis found [here](https://archive.ics.uci.edu/ml/machine-learning-databases/00468/online_shoppers_intention.csv). Each row in the data set contains pageview and session information belonging to a different visitor that browsed [Columbia](https://www.columbia.com.tr), an online e-commerce platform based in Turkey. Each row also contains the target class `REVENUE`, a boolean flag indicating whether that user session contributed to revenue or not. Pageview data includes the type of pages that the visitor browsed to and the duration spent on each page. Session information includes visitor-identifying data such as browser information, operating system information as well visitor location and type.
+
```

Comment:
My task is to describe the data set in the proposal, but now the proposal is just an archive of the old README, I will need to update this final report with the dataset description (adding a table of feature names, types, what information they explain) and variables that are important for the prediction.